### PR TITLE
David/backup state

### PR DESCRIPTION
**Pull request by druiz0992**
## What does this implement/fix? Explain your changes.
This PR imports and exports nightfall state

## Does this close any currently open issues?
Yes, #1060 

## What commands can I run to test the change? 
To export state (only works for geth) :
```
./bin/export-nightfall <folder>
```
State will be exported to `backup/folder`

To import state:
```
./bin/geth-standalone -i <FOLDER>
./bin/start-nightfall -l -d
./bin/import-nightfall <FOLDER>
```
`FOLDER` is the folder inside `backup` where state was exported
## Any other comments?
- Moved scripts to `.bin/`

### CODE REVIEW

File: export-nightfall [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+
+usage()
```
Comment (signorecello) [inline export-nightfall:3]: Hey, can these scripts go into the test folder, or even maybe create a new folder just for scripts? They start piling up on the root folder

**Review by signorecello — CHANGES_REQUESTED** [review]

File: export-nightfall (hunk 1 above)
Comment (druiz0992) [reply export-nightfall]: Done

**Review by israelboudoux — APPROVED** [review]
Nice work!

File: .github/workflows/check-PRs.yml [hunk 2]
```diff
@@ -386,25 +386,6 @@ jobs:
           name: ping-pong-test-logs
           path: ./ping-pong-test.log
 
-      - name: Export/Import state
```
Comment (signorecello) [inline .github/workflows/check-PRs.yml]: Sorry for reviewing this again. These tests look important, why are you removing them?

**Review by signorecello — CHANGES_REQUESTED** [review]

File: .github/workflows/check-PRs.yml (hunk 2 above)
Comment (druiz0992) [reply .github/workflows/check-PRs.yml]: I wasnt able to make this work on GHA

**Comment by israelboudoux** [comment]
You could move the bash files `start-apps` and `liquidity-provider` to the `bin` folder.

**Review by luizoamorim — APPROVED** [review]

**Pull request merged**